### ISSUE TITLE
BetterThemes

### ISSUE THREAD

**Issue by soulfireneedusername** [Vencord/plugin-requests#410]
### Plugin Description

Easily add buttons to switch themes to personalize your Discord interface. You can also create these buttons yourself and even add images to them. There is a button to scale those buttons, edit them to add your own CSS rules for personalization. This way, all your favourite themes will be saved in the themes tab and are extremely easy to switch. You can also use the Themes Diashow function and set it to a custom interval to automatically change any theme. If you want to add a theme or browse online themes quickly, you can just go to the online themes tab to browse through a preselection of the most popular themes of BetterDiscord Themes. This way, you don’t even have to add a picture or copy any links.

### Existing Plugin for other mod

https://betterdiscord.app/plugin/ThemeRepo

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Faf4a** [comment]
99% sure that this is a duplicate of #344

**Comment by soulfireneedusername** [comment]
Yes, I haven't seen that. I will probably focus more on local theme management, but I think online themes should replace the online themes tab. I also know that this shouldn't be a plugin.
